packages/app-store/stripepayment/extensions/EventTypeAppCard.tsx

```diff
@@ -61,7 +61,7 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
                 className="block w-full rounded-sm border-gray-300 pl-2 pr-12 text-sm"
                 placeholder="Price"
                 onChange={(e) => {
-                  setAppData("price", e.target.valueAsNumber * 100);
```

Comment:
@alannnc why is `valueAsNumber` not used anymore? This is from [RHF](https://react-hook-form.com/api/useform/register/#options) btw

Reply:
Probably something changed, valueAsNumber was being picked up correctly.